<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QMainWindow" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main_Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 44, 52);</string>
  </property>
  <widget class="QWidget" name="Central_Widget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 34, 39);
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="Stacked_Widget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Main_Menu_Widget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="title">
          <property name="font">
           <font>
            <pointsize>45</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(220, 225, 232);</string>
          </property>
          <property name="text">
           <string>Cybercraft</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="rubber_ducky_button">
          <property name="minimumSize">
           <size>
            <width>419</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>26</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(209, 213, 218);
}</string>
          </property>
          <property name="text">
           <string>Rubber Ducky</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="wifi_jammer_button">
          <property name="minimumSize">
           <size>
            <width>419</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>26</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(209, 213, 218);
}</string>
          </property>
          <property name="text">
           <string>Wi-Fi Jammer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="man_in_the_middle_button">
          <property name="minimumSize">
           <size>
            <width>419</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>26</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(209, 213, 218);
}</string>
          </property>
          <property name="text">
           <string>Man-in-the-Middle</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="war_driving_button">
          <property name="minimumSize">
           <size>
            <width>419</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>26</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(209, 213, 218);
}</string>
          </property>
          <property name="text">
           <string>War Driving</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Rubber_Ducky_Widget">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>280</y>
          <width>291</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(40, 44, 52);
}</string>
        </property>
        <property name="text">
         <string>Rubber Duck</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="WiFi_Jammer_Widget">
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>310</y>
          <width>291</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(40, 44, 52);
}</string>
        </property>
        <property name="text">
         <string>WiFi Jammer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Man_in_the_Middle_Widget">
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>300</y>
          <width>351</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(40, 44, 52);
}</string>
        </property>
        <property name="text">
         <string>Man in the Middle</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="War_Driving_Widget">
       <widget class="QTreeView" name="network_list">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>21</y>
          <width>441</width>
          <height>281</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>441</width>
          <height>191</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Sans</family>
          <pointsize>16</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="Tool_Bar">
   <property name="windowTitle">
    <string>Tool_Bar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	margin: 0px 10px;
}</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>34</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_home"/>
   <addaction name="action_help"/>
  </widget>
  <action name="action_home">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/breeze-icons/places/16/user-home.svg</normaloff>:/icons/breeze-icons/places/16/user-home.svg</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="action_help">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/breeze-icons/actions/16/help-about.svg</normaloff>:/icons/breeze-icons/actions/16/help-about.svg</iconset>
   </property>
   <property name="text">
    <string>Help_Documentation</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
